src/shared/src/components/AltinnInput.tsx

```diff
@@ -92,8 +92,7 @@ function AltinnInput(props: IAltinnInputProps) {
         className={validationError ? classes.altinnInputValidationError : classes.altinnInput}
       >
       {iconString ?
-        // eslint-disable-next-line jsx-a11y/click-events-have-key-events,jsx-a11y/no-static-element-interactions
-        <i data-testid="altinninput-iconString" className={`${classes.altinnInputIcon} ${iconString}`} onClick={focusInput}/> :
```

Comment:
Do we not need the click handler here to focus the input?

Reply:
My understanding was that it was redundant as it is also on the surrounding grid on line 74?

Reply:
Ah, that part was not visible by default in the PR changes tab, so didnt notice that. Could very well be that this is covered by the grid parent.